pipelines/h5c/rtp/v1/stage_1_task_scripts/do_REDCAL_KNOWN_GOOD.sh

```diff
@@ -72,26 +72,4 @@ cmd="redcal_run.py ${fn} \
                    --verbose"
 echo $cmd
 $cmd
-echo Finished running redcal at $(date)
-
-# add data products to librarian if desired
-if [ "${upload_to_librarian}" == "True" ]; then
-    if [ "${librarian_redcal_known_good}" == "True" ]; then
-        # get the integer portion of the JD
-        jd=$(get_int_jd ${fn})
-
-        # upload files to librarian
-        declare -a exts=(
-            ".known_good.first.calfits"
-            ".known_good.omni.calfits"
-            ".known_good.omni_vis.uvh5"
-            ".known_good.redcal_meta.hdf5"
-        )
-        for ext in ${exts[@]}; do
-            fn_out=`echo ${fn%.uvh5}${ext}`
-            echo librarian upload local-rtp ${fn_out} ${jd}/${fn_out}
-            librarian upload local-rtp ${fn_out} ${jd}/${fn_out}
-            echo Finished uploading ${fn_out} to the Librarian at $(date)
-        done
-    fi
-fi
+echo Finished running redcal at $(date)
```

Comment:
It's good practice to have a newline at the end of the last line.

Reply:
I like good practices! Did you change it already? I have added this on my local copy

Reply:
I didn't make any changes, just comments on github